clang_tidy/__init__.py

```diff
@@ -2,13 +2,15 @@
 import sys
 from pathlib import Path
 import functools
-import pkg_resources
+from importlib.resources import files
```

Comment:
```suggestion
try:
    from importlib.resources import files
except ImportError:
    from pkg_resources import resource_filename as files
```

might be a way to handle this then. We're not set-up to test the produced wheel against multiple versions atm. Testing happens through cibuildwheel, which uses Python 3.8 as per the `pyproject.toml`

3.8 has 9 months left before EOL, I'm hesitant to bump our minimum before that. @dokempf ?

Reply:
I made it check sys.version_info instead, since pkg_resources.resource_filename and importlib.resources.files have incompatible arguments.